Build error on HTML5 related to mbedTLS

**Godot version:** Git 06a42123779fd3c0f3118ddbd8ed393861f25ee3

**OS/device including version:** Fedora 30, Emscripten `sdk-releases-fastcomp-0ca6c49e30bc09c3ccefc867df4196a4b4183441-64bit`

**Issue description:** Building for HTML5 fails with the following error:

```text
Compiling ==> thirdparty/mbedtls/library/aes.c
thirdparty/mbedtls/library/aes.c:31:10: error: expected "FILENAME" or <FILENAME>
#include MBEDTLS_CONFIG_FILE
         ^
<command line>:15:29: note: expanded from here
#define MBEDTLS_CONFIG_FILE thirdparty/mbedtls/include/godot_core_mbedtls_config.h
                            ^
1 error generated.
shared:ERROR: compiler frontend failed to generate LLVM bitcode, halting
scons: *** [thirdparty/mbedtls/library/aes.javascript.opt.bc] Error 1
scons: building terminated because of errors.
```

[Full build log on GitLab CI](https://gitlab.com/Calinou/godot-builds-ci/-/jobs/246873655)

**Steps to reproduce:** Build for HTML5 using `scons platform=html5 tools=no target=release`.

--- a/core/math/SCsub
+++ b/core/math/SCsub
@@ -22,7 +22,7 @@ if not has_module:
     env_thirdparty = env_math.Clone()
     env_thirdparty.disable_warnings()
     # Custom config file
-    env_thirdparty.Append(CPPDEFINES=[('MBEDTLS_CONFIG_FILE', "thirdparty/mbedtls/include/godot_core_mbedtls_config.h")])
+    env_thirdparty.Append(CPPDEFINES=[('MBEDTLS_CONFIG_FILE', '\\"thirdparty/mbedtls/include/godot_core_mbedtls_config.h\\"')])
     thirdparty_mbedtls_dir = "#thirdparty/mbedtls/library/"
     thirdparty_mbedtls_sources = [
         "aes.c",

